README.adoc

```diff
@@ -219,23 +224,23 @@ acceptance tests. They can be run via:
 $ bundle exec rake features
 ---------------------------
 
-_NOTE_: Only Linux OS is supported at present.
+*NOTE:* These tests can only be executed in Linux, at present.
```

Comment:
Well, then we should say, that the test cannot be run on Windows, referencing the issue. But saying that they only run on Linux is not true.